boot/freeldr/freeldr/arch/drivers/hwide.c

```diff
@@ -653,41 +696,29 @@ IdentifyDevice(
 
     /* Send the IDENTIFY DEVICE command */
     AtaWritePort(Channel, IDX_IO1_o_Command, IDE_COMMAND_IDENTIFY);
-    StallExecutionProcessor(50);
+    StallExecutionProcessor(500);
```

Comment:
why?

Reply:
For slow devices.

Reply:
Ok, I reverted the change (seems like unnecessary).